openapi.yml

```
@@ -38,6 +38,32 @@ servers:
           - 'http://localhost:3000/api/v3'
         default: 'https://tractionguest.ca/api/v3'
 paths:
+  '/batches/{batch_id}':
+    parameters:
+      - schema:
+          type: string
+        name: batch_id
+        in: path
+        required: true
+    get:
+      summary: Get a BatchJob
+      tags:
+        - Batches
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: ./models/BatchJob.v1.yaml
+        '404':
```

You'll probably want to change this to a wildcard:

```suggestion
        '4XX':
```